app/assets/javascripts/new_idea.js

```diff
@@ -0,0 +1,18 @@
+var addNewIdea = function(requestService) {
+  $('.new-idea').on("click", "#ideaSave", function(event){
+    event.preventDefault()
+    var idea = { title: $('#newTitle').val(), body: $('#newBody').val()}
```

Comment:
You may want to consider storing the selectors in variables so that you don't keep requerying for them.